.github/workflows/release.yaml

```diff
@@ -0,0 +1,29 @@
+name: Build & publish
+
+# TODO: Setup release workflow
+on:
+  push:
+    branches: [ "**" ]
+    tags:
+    - "client-v*"
+#  pull_request:
+#    branches: [main]
+
+jobs:
+  deploy-client:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./client
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/client-')
+    strategy:
+      matrix:
+        python-version: [3.7]
```

Comment:
by the way, @kindermax, what minimun version we should support? can we start from 3.9/3.10 at least? maybe do a poll in dev channel?